coalib/settings/ConfigurationGathering.py

```diff
@@ -147,7 +147,15 @@ def load_configuration(arg_list, log_printer, arg_parser=None):
     for item in list(cli_sections["default"].contents.pop("targets", "")):
         targets.append(item.lower())
 
-    if bool(cli_sections["default"].get("no_config", "False")):
+    # list of cli options which do not involve load config file
+    no_cofig_file_options = (
+        'no_config',
+        'show_bears',
+        'show_capabilities')
+
+    if any(
+            option in cli_sections["default"]
+            for option in no_cofig_file_options):
```

Comment:
Okay. I said because i saw it in the vim and it showed exactly 79 characters.  See the below screenshot below to let me know if i am missing something.

![capture-13](https://cloud.githubusercontent.com/assets/1281636/18829069/47c71360-83f8-11e6-8c4c-bfb97e82af3f.PNG)